### PR TITLE
Adds ability to pass through []string key

### PR DESCRIPTION
**Pull request by IRCody**
Utilizes new proto field to pass through []string key so that `.` can
be used in namespaces.

Depends on [#1248](https://github.com/intelsdi-x/snap/pull/1248) being merged into snap.

### CODE REVIEW

File: v1/plugin/config_policy.go [hunk 1]
```diff
@@ -61,7 +61,10 @@ func (c *ConfigPolicy) AddNewIntRule(ns []string, key string, req bool, opts ...
 	}
 	k := strings.Join(ns, ".")
 	if c.integerRules[k] == nil {
-		c.integerRules[k] = &rpc.IntegerPolicy{Rules: map[string]*rpc.IntegerRule{}}
+		c.integerRules[k] = &rpc.IntegerPolicy{
```
Comment (kindermoumoute) [inline v1/plugin/config_policy.go]: Don't we want `c.integerRules[k]` to be a slice instead?

Comment (IRCody) [inline v1/plugin/config_policy.go]: No. It uses this format to maintain backward compatibility.

**Comment by kindermoumoute** [comment]
LGTM

**Comment by kindermoumoute** [comment]
Still Good To Me.

**Pull request merged**